### PR TITLE
x11-misc/*: adopt + cleanup five small packages

### PR DESCRIPTION
**Pull request by ionenwks**
Notes:
- numlockx was EAPI-7 bumped in stable and would rather not revbump to ~arch (but will if asked)
- wmctrl also received changes in stable but very trivial (hardly needed attention, but picking up since using in some old scripts).
- rest already had ~arch versions, so only changed those

### CODE REVIEW

File: x11-misc/dzen/dzen-0.9.5_p2.ebuild [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	emake -C gadgets \
+		CC="$(tc-getCC)" CFLAGS="${cflags}" \
+		LDFLAGS="${LDFLAGS}" X11LIBS="${libs}"
+
+	local flag
+	for flag in xft xinerama xpm; do
+		# xft always-enabled wrt bug #477656
+		[[ ${flag} == xft ]] || use ${flag} || continue
+		cflags+=" $($(tc-getPKG_CONFIG) --cflags ${flag}) -DDZEN_${flag^^}"
+		libs+=" $($(tc-getPKG_CONFIG) --libs ${flag})"
```
Comment (SoapGentoo) [inline x11-misc/dzen/dzen-0.9.5_p2.ebuild:53]: let's say `USE="-xinerama"`, this will still unconditionally try to add the libs from `xinerama.pc`?

Comment (SoapGentoo) [reply x11-misc/dzen/dzen-0.9.5_p2.ebuild:53]: scratch that, I for the logic wrong. Could you maybe consider de-tersifying the continue?
```
if [[ ${flag} == xft ]] || use ${flag}; then
    continue
fi

File: x11-misc/dzen/dzen-0.9.5_p2.ebuild [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	done
```
Comment (SoapGentoo) [inline x11-misc/dzen/dzen-0.9.5_p2.ebuild:54]:
```
	local flag
	# xft always-enabled wrt bug #477656
	for flag in xft $(usev xinerama) $(usev xpm); do
		cflags+=" $($(tc-getPKG_CONFIG) --cflags ${flag}) -DDZEN_${flag^^}"
		libs+=" $($(tc-getPKG_CONFIG) --libs ${flag})"
	done
```

Comment (ionenwks) [reply x11-misc/dzen/dzen-0.9.5_p2.ebuild:50]: Done, much better.

**Review by SoapGentoo — APPROVED** [review]

**Pull request closed without merging**